app/config/parameters.yml.dist

```diff
@@ -19,16 +19,18 @@ parameters:
     database_path: "%kernel.root_dir%/../data/db/wallabag.sqlite"
     database_table_prefix: wallabag_
     database_socket: null
+    # with MySQL, use "utf8mb4" if got problem with content with emojis
+    database_charset: utf8
```

Comment:
You set `utf8` as a default value (and the default database storage is SQLite) but you put `utf8mb4` in `parameters_test.sqlite.yml`. Are you sure?.